### PR TITLE
Clarify snaps deployment

### PR DESCRIPTION
**Pull request by BanzaiMan**

### CODE REVIEW

File: user/deployment/snaps.md [hunk 1]
```diff
@@ -15,34 +15,51 @@ dist: xenial
...
 ```
 
 _Note: The final `-` requests the login be exported to stdout instead of a file. It is required._
 
-The token will be printed out. Copy and put it into the Travis CI environment variable `$SNAP_TOKEN`:
+The token will be printed out. 
+
+_Note: The `edge` channel is intended for the bleeding edge: your every commit to master will be built and uploaded._
+
+### Using the CLI cleint
```
Comment (MariadeAnton) [inline user/deployment/snaps.md]:
```suggestion
### Using the CLI client
```

**Comment by MariadeAnton** [comment]
Thank you @BanzaiMan!

File: user/deployment/snaps.md [hunk 2]
```diff
@@ -15,34 +15,51 @@ dist: xenial
...
-  snap: my.snap
+  snap: my_*.snap
   channel: edge
   skip_cleanup: true
 ```
 {: data-file=".travis.yml"}
 
-To upload snaps from Travis CI, export a Snap Store login token. You can do this with the snapcraft command-line tool, once you have [enabled snap support](https://docs.snapcraft.io/core/install) on your system.
+The `snap` value should be a string that matches exactly one file when the deployment starts.
+If the name of the snap file is not known ahead of time, you can use a shell glob pattern, as shown
```
Comment (MariadeAnton) [inline user/deployment/snaps.md]:
```suggestion
If the name of the snap file is not known ahead of time, you can use a shell glob pattern, as shown in the example above.
```

File: user/deployment/snaps.md [hunk 3]
```diff
@@ -15,34 +15,51 @@ dist: xenial
...
+in the example above.
```
Comment (MariadeAnton) [inline user/deployment/snaps.md]:
```suggestion

```

File: user/deployment/snaps.md [hunk 4]
```diff
@@ -15,34 +15,51 @@ dist: xenial
...
 
 _Note: The final `-` requests the login be exported to stdout instead of a file. It is required._
 
-The token will be printed out. Copy and put it into the Travis CI environment variable `$SNAP_TOKEN`:
+The token will be printed out. 
+
+_Note: The `edge` channel is intended for the bleeding edge: your every commit to master will be built and uploaded._
+
+### Using the CLI cleint
+Using our [CLI client](https://github.com/travis-ci/travis.rb#readme), define `$SNAP_TOKEN`:
```
Comment (MariadeAnton) [inline user/deployment/snaps.md]:
```suggestion
Using our Travis CI, [CLI client](https://github.com/travis-ci/travis.rb#readme), define `$SNAP_TOKEN`:
```

File: user/deployment/snaps.md [hunk 5]
```diff
@@ -15,34 +15,51 @@ dist: xenial
...
 
 ```bash
+# in the repository root
 travis env set SNAP_TOKEN "<token>"
 ```
 
-_Note: The `edge` channel is intended for the bleeding edge: your every commit to master will be built and uploaded._
+### Using Settings page
+Equivalently, you can do this on the [Settings page](https://docs.travis-ci.com/user/environment-variables#defining-variables-in-repository-settings).
```
Comment (MariadeAnton) [inline user/deployment/snaps.md]:
```suggestion
Equivalently, you can do this on the [Settings page](https://docs.travis-ci.com/user/environment-variables#defining-variables-in-repository-settings) of your repository at Travis CI.
```

File: user/deployment/snaps.md (hunk 2 above)
Comment (BanzaiMan) [reply user/deployment/snaps.md]: This newline is unnecessary.

File: user/deployment/snaps.md (hunk 4 above)
Comment (BanzaiMan) [reply user/deployment/snaps.md]: A good idea, but the comma is superfluous.

**Comment by MariadeAnton** [comment]
Thanks for updating so swiftly @BanzaiMan, I think we're ready to merge this in as soon as the PR build finishes :)

**Pull request merged**